lib/simple_form/error_notification.rb

```diff
@@ -0,0 +1,11 @@
+module SimpleForm
+  class ErrorNotification
```

Comment:
> Can we drop a comment describing that is a monkeypatch and not us defining our own class? Otherwise it looks like we're defining a class that's missing a bunch of other methods

Yep, good call. Added in ce8f261.